src/main/webapp/js/chart-loader.js

```diff
@@ -0,0 +1,44 @@
+
+//Loads and initializes the Google Charts API, then calls drawChart()
+function initGoogleCharts() {
+    google.charts.load('current', {packages: ['corechart']});
+    google.charts.setOnLoadCallback(drawChart);
+}
+
+//Sets up chart data in a DataTable, draws a chart on the page
+function drawChart(){
+    var foodData = new google.visualization.DataTable();
+
+    foodData.addColumn("string", "Food Title");
+    foodData.addColumn("number", "Carbon Footprint");
```

Comment:
kind of a nitpick, but it might be useful to add in a unit for the carbon footprint (ex "Carbon Footprint (kg of CO2 emitted) or alternatively just mention somewhere on the page what the carbon footprint unit is. Not really a code thing, just something that would help the user gain context/info.